projects/plugins/jetpack/json-endpoints/class.wpcom-json-api-upload-media-v1-1-endpoint.php

```diff
@@ -93,6 +93,18 @@ public function callback( $path = '', $blog_id = 0 ) {
 		// We're splitting out videos for Jetpack sites.
 		foreach ( $media_files as $media_item ) {
 			if ( preg_match( '@^video/@', $media_item['type'] ) && $is_jetpack_site ) {
+				if ( defined( 'IS_WPCOM' ) && IS_WPCOM &&
+					defined( 'VIDEOPRESS_JETPACK_VIDEO_ENABLED' ) && VIDEOPRESS_JETPACK_VIDEO_ENABLED
+				) {
+					// Check that video upload space is available for a Jetpack site (skipped if site is Atomic).
+					if ( function_exists( 'videopress_filter_jetpack_get_space_used' ) ) {
+						$result = videopress_check_space_available_for_jetpack( $blog_id, $media_item['name'], $media_item['size'] );
+					}
+
+					if ( true !== $result ) {
+						$this->api->output_early( 400, array( 'errors' => $this->rewrite_generic_upload_error( array( $result ) ) ) );
+					}
```

Comment:
> We probably don't need the function_exists check here, since we already guard this with IS_WPCOM where the function always exists.

Isn't this the same case with the check on 122 that I just added back? That's also wrapped in a `if ( defined( 'IS_WPCOM' ) && IS_WPCOM ) {` gate right?

Reply:
You're right! I missed that. :) So go ahead and remove it again there, sorry for the back-and-forth.

That other one still needs the missing `remove_filter` call added though, don't forget to do that part.

Reply:
Removed the check and added the remove_filter!